deployment/legacy-migration/maintenance-worker/migration/mongo/.env

```diff
@@ -12,6 +12,6 @@
 # On Windows this resolves to C:\Users\dornhoeschen\AppData\Local\Temp\mongo-export (MinGW)
 EXPORT_PATH='/tmp/mongo-export/'
 EXPORT_MONGOEXPORT_BIN='mongoexport'
-MONGO_EXPORT_SPLIT_SIZE=6000
```

Comment:
Code creep?

Reply:
good call, I had seen this and wanted to revert the change, but then forgot... 
thanks for calling me out!! we certainly want to avoid code creep!